README.md

```diff
@@ -56,6 +56,10 @@
         <a href="https://semaphore.pse.dev/discord">
             🗣️ Chat &amp; Support
         </a>
+        <span>&nbsp;&nbsp;|&nbsp;&nbsp;</span>
```

Comment:
There was no link to `js.semaphore.pse.dev` anywhere. Was it intentional?
If yes and the intention is to not use the typedocs anymore, we could delete the corresponding GH workflows, npm script and config file.

Reply:
i remember that typedocs is one of the things we'd like to keep for Semaphore at 99% but circle back to @cedoor or @vplasencia for more details!

Reply:
Hey @sripwoud 

Typedoc is very important for the project. There is a typedoc link in every package with type documentation. I agree that it would be better to have it in the main Readme file as well. 

Should it be part of the end of the Packages section instead of the beginning of the file? What do you all think?

Reply:
What about both? in the main README and in the packages' READMEs?
In any case I personally prefer to see such a doc link at the top of a README: the more I have to scroll down, the more likely I'll miss that link :wink: (I had missed the typedoc links [here](https://github.com/semaphore-protocol/semaphore/blob/main/packages/group/README.md#-usage)).
I would also add it somewhere on https://docs.semaphore.pse.dev/

Reply:
Makes sense. Thanks. Feel free to create an issue to add it to https://docs.semaphore.pse.dev/